src/components/userFeedback/UserFeedback.spec.tsx

```diff
@@ -0,0 +1,143 @@
+import { shallow, ShallowWrapper } from 'enzyme';
```

Comment:
With multiple html elements nested inside a component, I think mount should be preferred most of the time, but that would still depends on your test case.